src/app/youtube/components/sort/sort.component.html

```diff
@@ -1,22 +1,42 @@
 <div class="sort-wrapper" [class.visible]="isVisible">
   <div class="container sort-container">
     <span>Sort by</span>
-    <button class="btn-primary" (click)="setSortType(1)" [ngClass]="{
+    <button
+      class="btn-primary"
+      (click)="setSortType(1)"
```

Comment:
use SortType enum here

Reply:
fixed using a reference to SortType enum in component